tools/compliance/sbom.bzl

```diff
@@ -27,6 +27,12 @@ def _sbom_impl(ctx):
     inputs = [inps[0]]
     args.add("--packages_used", inps[0].path)
     args.add("--out", ctx.outputs.out.path)
+    if ctx.attr._maven_install:
```

Comment:
won't this always be true b/c it has a default set?

Reply:
Sigh. Yes, but I wanted the ability to be able to run this in test most without providing the lock file.